models/hh_cond_exp_traub_implicit.nestml

```diff
@@ -64,16 +64,16 @@ neuron hh_cond_exp_traub_implicit:
 
     # equilibrium values for (in)activation variables
     function V_rel mV = V_m - V_T
-    function alpha_n real = 0.032 * ( 15. - V_rel ) / ( exp( ( 15. - V_rel ) / 5. ) - 1. )
-    function beta_n real = 0.5 * exp( ( 10. - V_rel ) / 40. )
-    function alpha_m real = 0.32 * ( 13. - V_rel ) / ( exp( ( 13. - V_rel ) / 4. ) - 1. )
-    function beta_m real = 0.28 * ( V_rel - 40. ) / ( exp( ( V_rel - 40. ) / 5. ) - 1. )
-    function alpha_h real = 0.128 * exp( ( 17. - V_rel ) / 18. )
-    function beta_h real = 4. / ( 1. + exp( ( 40. - V_rel ) / 5. ) )
+    function alpha_n real = 0.032 * ( 15. - V_rel / 1 mV ) / ( exp( ( 15. - V_rel / 1 mV ) / 5. ) - 1. )
+    function beta_n real = 0.5 * exp( ( 10. - V_rel / 1 mV ) / 40. )
+    function alpha_m real = 0.32 * ( 13. - V_rel / 1 mV ) / ( exp( ( 13. - V_rel / 1 mV ) / 4. ) - 1. )
+    function beta_m real = 0.28 * ( V_rel / 1 mV - 40. ) / ( exp( ( V_rel / 1 mV - 40. ) / 5. ) - 1. )
+    function alpha_h real = 0.128 * exp( ( 17. - V_rel / 1 mV ) / 18. )
+    function beta_h real = 4. / ( 1. + exp( ( 40. - V_rel / 1 mV ) / 5. ) )
 
-    Act_m' = alpha_m - ( alpha_m + beta_m ) * Act_m
-    Act_h' = alpha_h - ( alpha_h + beta_h ) * Act_h
-    Inact_n' = alpha_n - ( alpha_n + beta_n ) * Inact_n
+    Act_m' = ( alpha_m - ( alpha_m + beta_m ) * Act_m ) * 1 [1/ms]
+    Act_h' = ( alpha_h - ( alpha_h + beta_h ) * Act_h ) * 1 [1/ms]
+    Inact_n' = ( alpha_n - ( alpha_n + beta_n ) * Inact_n ) * 1 [1/ms]
```

Comment:
Similar to above, but I find the notation here more confusing; `/ 1 ms` would probably be easier, but I think no unit manipulation should happen here at all.